Sources/SwiftParser/Availability.swift

```diff
@@ -253,6 +253,28 @@ extension Parser {
     return self.consumeAnyToken()
   }
 
+  /// Consume the unexpected version token(e.g. integerLiteral, floatingLiteral) until the period no longer appears.
+  private mutating func parseUnexpectedVersionTokens(unexpectedTokens: [RawSyntax] = []) -> RawUnexpectedNodesSyntax? {
```

Comment:
I don’t think there’s a need to pass `unexpectedTokens` in here. This can just return `RawUnexpectedNodesSyntax?` and then you can combine the two unexpected lists in the function that calls this.

Reply:
Oh, there was a useful initializer that combines RawUnexpectedNodesSyntaxs! I've made the modification to use it.